interface/src/app/gene/network/network.js

```diff
@@ -5,6 +5,7 @@
 angular.module('adage.gene.network', [
   'ui.router',
   'ui.bootstrap',
+  'ngResource',
```

Comment:
Angular didn't complain before because `ngResource` has been included in other modules. I noticed this issue when writing unit tests for `download` module.